src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3062,6 +3073,11 @@ enum PlanAccessMode {
   """
   BICYCLE_RENTAL
   """
+  Driving a car from the origin to the destination.
+  TODO add more thorough description
```

Comment:
Please resolve these TODOs.

Reply:
TODOs are now resolved, the comments are based on the assumption that the access, egress, and transfer modes should all be equal when one is set to `CAR`. Once the issue brought up by Henrik is solved this might have to be changed